pandas/core/resample.py

```diff
@@ -239,7 +239,9 @@ def aggregate(self, arg, *args, **kwargs):
         self._set_binner()
         result, how = self._aggregate(arg, *args, **kwargs)
         if result is None:
+            grouper = kwargs.get('grouper', None)
```

Comment:
What happens if the user passes a function like `def f(arr, grouper)`? Does this raise? If so, can we provide a good error message?

Reply:
This is correct. Is it okay to pass `None` like?:
```
self._groupby_and_aggregate(arg, None, *args, **kwargs)
```